src/v2/Apps/Consign/consignRoutes.tsx

```diff
@@ -18,4 +18,21 @@ export const consignRoutes = [
       }
     `,
   },
+  {
+    path: "/consign/:submissionID/offer/:offerID",
```

Comment:
The other app in this routes file falls under `/consign2/...`, but since this an unused URL I opted to use `/consign/`, saving us having to change it once the new consign app is live.

...though I just realized that I left `/submission/` out of the path, so I'll change this path to `/consign/submission/:submissionID/offer/:offerID` and verify that's not a used path in the existing consignment submission form.

Reply:
Ooh good call about not conflicting with the existing routing.

If so might be the path of least resistance to do `/consign/offer/:offerID`. 🤷

Reply:
hrrmmmm, I don't think it's a conflict in terms of existing routes, but the old app _does_ redirect `/consign/submission/* ` into the old form: https://github.com/artsy/force/blob/edfad32e449a76a5e5e69e33bb5e683654b3db95/src/desktop/apps/consign/index.js#L35

I'm just going to use `/consign/offer/:offerID` to avoid having to add an ignore-list to https://github.com/artsy/force/blob/edfad32e449a76a5e5e69e33bb5e683654b3db95/src/desktop/apps/consign/routes.ts#L56 😅 

(For followers unfamiliar, we don't _need_ the submissionID to get the offer information. Sarah and I were just thinking it made sense to include it in the URL, from a restful perspective.)